packages/react-components/src/components/MessageThread.tsx

```diff
@@ -986,7 +986,13 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
         }
       }
     }
-  }, [numberOfChatMessagesToReload, onLoadPreviousChatMessages]);
+  }, [
+    numberOfChatMessagesToReload,
+    onLoadPreviousChatMessages,
+    isLoadingChatMessagesRef,
```

Comment:
Just a note that adding these Refs to the use* dependency array shouldn't impact any bug, the ref from `useRef` cannot change (so its fine to include it but it should be the same behavior as not including it)

Reply:
You're right, reverted these changes and pushed the update for the PR